src/hooks/use-chat-context.tsx

```diff
@@ -142,11 +140,7 @@ export const ChatProvider: React.FC< ChatProviderProps > = ( { children } ) => {
 
 	useWindowListener( 'focus', async () => {
 		// When the window is focused, we need to kick off a request to refetch the theme details, if server is running.
-		if (
-			! selectedSite?.id ||
-			selectedSite.running === false ||
-			selectedSite?.isAddingSite === true
-		) {
+		if ( ! selectedSite?.id || selectedSite.running === false || ! selectedSite?.isAddingSite ) {
```

Comment:
I think the condition `! selectedSite?.isAddingSite` should be the opposite, `selectedSite?.isAddingSite`. I.e. when the app gains focus, only fetch plugins and themes for sites that are running and not being added.

Otherwise, changing themes/plugins on a running site are not reflected in the chat context.

Reply:
:man_facepalming: . Updated!